components/release/base/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+resources:
+- argocd-roles
+- release-pipeline-role
```

Comment:
Why is there a directory `release-pipeline-role` containing a single yaml in it and a `kustomization` file instead of just pointing to the yaml file here? Same question for `authentication`

Reply:
do you mean to say they should be in `1 folder` with `1 kustomization.yaml` file?

> I am following what the `has` team has been using up to now. See https://github.com/redhat-appstudio/infra-deployments/tree/main/components/has/base

or do you mean to say why are we even using `kustomization.yaml` files anyways?

> I had issues including them in overlays without a `kustomization.yaml` file

Reply:
I am saying why not replace `release-pipeline-role` with `release-pipeline-resources-clusterrole.yaml` and move the yaml up a dir and not have the extra `release-pipeline-role` dir

Reply:
> I am saying why not replace `release-pipeline-role` with `release-pipeline-resources-clusterrole.yaml` and move the yaml up a dir and not have the extra `release-pipeline-role` dir

I can do that!

Reply:
fixed.

Reply:
awesome, thanks